test-suite/target/bin/luajit: ffi_include_std.lua:42: unexpected symbol near '__nmatch' at line 5287
stack traceback:
        [C]: in function 'cdef'
        ffi_include_std.lua:42: in main chunk
        [C]: at 0x55c8e5bc5e90
Failed test when running /home/apkbuild/alpaquita-aports2/universe/luajit/src/luajit2-2.1-20230410/test-suite/target/bin/luajit ffi_include_std.lua 1: 256

1 tests failed.
--- a/test-suite/test/sysdep/ffi_include_std.lua
+++ b/test-suite/test/sysdep/ffi_include_std.lua
@@ -23,7 +23,6 @@
 #include <complex.h>
 #include <elf.h>
 #include <mqueue.h>
-#include <regex.h>
 #include <fcntl.h>
 ]]
   fp:close()
